src/GRA.Controllers/HomeController.cs

```diff
@@ -222,6 +223,16 @@ var schedulingStage
                         .GetUpcomingStreamListAsync()
                 };
 
+                var latestBookCookie = Request.Cookies[$"{GetActiveUserId()}{LatestBook}"];
+                if (latestBookCookie != null)
+                {
+                    var title = latestBookCookie.Split(";")[0];
```

Comment:
Did you test this if the string doesn't contain `;`? I suspect it reacts....poorly. Maybe check if `.Contains(";")` and if not assume it's title only?

Reply:
Oh I see below that you always put a `;` even if there is no author. I'd still add the `.Contains()` here as it's an exception if you split on a character that's not present I think.

Reply:
I just tested it and Split returns an array of size 1 containing the original string if the split character is not present.

Reply:
Just pushed a fix. If you split on a character that isn't present the return is an array of length 1 with the original string, but that would still have caused an error when trying to access the Author from cookie[1].